main.js

```diff
@@ -1 +1,119 @@
+const API_URL = 'https://opentdb.com/api.php?amount=10&category=9&difficulty=medium&type=multiple';
+const START_BUTTON = document.getElementById('startBtn');
```

Comment:
大文字のスネークケース(`_` で連結する)にする場合は定 `数` の時だけかと思います。
`API_URL` は定 `数` なのでいいのですが、これ以降は定数ではないので、以下のように書いた方がいいと思います。
https://qiita.com/satotaka/items/21da1ccc0171df85d053

```suggestion
const startButton = document.getElementById('startBtn');
```